### ISSUE TITLE
Error when comment on first line of inserted file

### ISSUE THREAD

**Issue by coemgenuslcp** [pml-lang/converter#47]
Version 1.5.0 2021-06-08

When I call `[insert file="other file.pml"]`, if the first line of the other file is a comment like so:
```
[- Here is a comment. -]
Here is a paragraph.
```
the converter errs with this error:
```
Error: '-' is an invalid node tag.
Valid tags are: [...].
[ >>> - Here is a comment. -]
line 1, column 2
```
If the comment is on later lines, there is no error.

**Comment by pml-lang** [comment]
Good catch! Thanks.

I was able to reproduce the bug in PML version 1.5.0.

But the example works correctly in PML version 2.0. (Note: PML 2.0 is currently under construction. It uses a new [pXML](https://pxml-lang.github.io/) parser).

I will close this issue after publishing PML 2.0.

**Comment by pml-lang** [comment]
This bug has been fixed in version 2.0.0
